webworker/index.ts

```diff
@@ -119,6 +119,8 @@ const webWorkerApi = {
       delete eventListeners[event]
     }
   },
+
+  kill: async () => {},
```

Comment:
it's confusing to have this here if it's never called, let's remove it or add an annotation indicating it's never called